controllers/tasks.js

```diff
@@ -51,7 +51,8 @@ const addNewTask = async (req, res) => {
  */
 const fetchTasks = async (req, res) => {
   try {
-    const allTasks = await tasks.fetchTasks();
+    const allTasksPromises = await tasks.fetchTasks();
```

Comment:
Didn't await make the code wait for fetch tasks?

Reply:
await is returning a list of promises.